jcommune-view/jcommune-web-view/src/test/java/org/jtalks/jcommune/test/utils/Groups.groovy

```diff
@@ -39,8 +41,8 @@ class Groups {
     @Autowired
     MockMvc mockMvc
 
-    boolean isExist(String username) {
-        return groupDao.getByName(username) != null
+    boolean isExist(String groupName) {
+        return !groupDao.getByName(groupName).empty
```

Comment:
I don't think so. Method `getByName` returns `List<Group>` so I check the list is not empty.